imap_processing/glows/l1b/glows_l1b_data.py

```diff
@@ -482,13 +480,17 @@ class HistogramL1B:
     number_of_bins_per_histogram: int
     number_of_events: int
     filter_temperature_average: np.double
-    filter_temperature_variance: np.double
+    filter_temperature_variance: InitVar[np.double]
+    filter_temperature_std_dev: np.double = field(init=False)
```

Comment:
Did you mean to keep both? Just want to make sure you intended to do that because others had only one.

Reply:
nvm, looks like your post unit uses these to calculate std_dev